teacher_module.rb

```diff
@@ -0,0 +1,36 @@
+module Teacher_stuff
```

Comment:
`Teacher` and `ApprenticeTeacher` are both Staff or Instructor or Educator or whatever you want to call it.  Since it's an is-a relationship, use inheritance.  Modules are best when the two things with similar behaviors don't have a common superclass.